src/components/AssetClassification/EntityDetailDisplay.tsx

```diff
@@ -0,0 +1,38 @@
+import { FunctionComponent, useEffect, useState } from "react";
+import { EntityDetail } from "../../models"
+import { InputOrDisplay } from "../Input";
+
+interface EntityDetailProps {
+    detail: EntityDetail,
+    editable: boolean,
+    handleChange: () => any,
+}
+
+const initialState = (entityDetail?: EntityDetail) => ({
+    name: entityDetail?.name || '',
+    description: entityDetail?.description || '',
+    home_url: entityDetail?.home_url || '',
+    source_url: entityDetail?.source_url || '',
+})
+
+export const EntityDetailDisplay: FunctionComponent<EntityDetailProps> = ({ detail, editable, handleChange }) => {
```

Comment:
Wasn't sure if I should name it just `EntityDetail` or not because there's already an interface with that name, but `FeeDestination.tsx` existing made me think maybe I should.  Thoughts?